vprdb/converters/kitti_converter.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import mrob
+import numpy as np
+import open3d as o3d
+import os
+from pathlib import Path
+import pykitti
+import shutil
```

Comment:
Now they are in alphabetical order, but let's divide `import ...` and `from ...` statements
```suggestion
import argparse
import mrob
import numpy as np
import open3d as o3d
import os
import pykitti
import shutil

from pathlib import Path
```